sentence_transformers/sparse_encoder/SparseEncoder.py

```diff
@@ -75,6 +75,109 @@ class SparseEncoder(SentenceTransformer):
 
     # -----------------------------------Added------------------------------------
     # - get_sparsity_stats
+    """
+    Loads or creates a SparseEncoder model that can be used to map sentences / text to sparse embeddings.
+
+    Args:
+        model_name_or_path (str, optional): If it is a filepath on disc, it loads the model from that path. If it is not a path,
+            it first tries to download a pre-trained SentenceTransformer model. If that fails, tries to construct a model
```

Comment:
```suggestion
            it first tries to download a pre-trained SparseEncoder model. If that fails, tries to construct a model
```
It's smart to ctrl+F "SentenceTransformer" on the docs page and to try and resolve those